docs/source/troubleshooting.rst

```diff
@@ -0,0 +1,41 @@
+Troubleshooting
```

Comment:
Does this page appear in the table of contents?

Reply:
Fixed.